.github/workflows/covid-scheduled-bot.yaml

```diff
@@ -32,6 +32,9 @@ jobs:
             - name: Get National Data From djay/covidthailand
               working-directory: bot_jobs
               run: python get_national_data.py
+            - name: Get and Build Province deaths
```

Comment:
Add death data to build_job.json generated by build_province_graph.py so all fields in the table can use one file

Reply:
Resolved